src/services/email.integration-test.ts

```diff
@@ -74,25 +110,22 @@ describe("Email service", () => {
 
       const current = await getMails();
 
-      const receivers = parseEmailToEnv(env.SMTP_TO);
```

Comment:
There is a normalization of `receivers` name into `recipients`, to ensure uniform naming across the codebase.